spec/refresher_spec.rb

```diff
@@ -75,8 +75,8 @@
 
   context "on loops, yeah!!!" do
     it "should loop over the array and return a new array" do
-      loopy = [2,3,4]
-      expect(loopy.each { |n| n + 1 }).to eq [2,3,4]
+      loopy = [1,2,3]
+      expect(loopy.map! { |n| n + 1 }).to eq [2,3,4]
```

Comment:
There you go!  Er, except you don't want the `!` in there.  That will modify the array in place, as opposed to simply returning the desired array, which is all that is needed here.  I'm not sure what your experience level is, but the general thinking is that, all else equal, it is better to avoid modifying objects in place.